sw/device/lib/dif/dif_rv_core_ibex.h

```diff
@@ -106,6 +106,21 @@ typedef enum dif_rv_core_ibex_error_status {
   kDifRvCoreIbexErrorStatusAll = (1 << 0 | 1 << 8 | 1 << 9 | 1 << 10),
 } dif_rv_core_ibex_error_status_t;
 
+typedef enum dif_rv_core_ibex_rnd_status {
+  /**
+   * The current rnd word is valid.
+   */
+  kDifRvCoreIbexRndStatusValid = 1 << 0,
+  /**
+   * The current rnd word is fips compliant.
+   */
+  kDifRvCoreIbexRndStatusFipsCompliant = 1 << 1,
+  /**
+   * The current rnd word is either valid and fips compliant.
+   */
+  kDifRvCoreIbexRndStatusAll = 0x03,
```

Comment:
Do we really need this constant? If we really do:

```suggestion
  kDifRvCoreIbexRndStatusAll = kDifRvCoreIbexRndStatusValid | kDifRvCoreIbexRndStatusFipsCompliant,
```

Reply:
Better remove it.